iotdb-client/client-py/iotdb/dbapi/Cursor.py

```diff
@@ -125,10 +129,49 @@ def execute(self, operation, parameters=None):
                     time_names = [
                         name for name in seq.replace("FROM Time Name", "").split()
                     ]
+                elif seq.find("GROUP BY") >= 0:
+                    time_group_sql = seq
+                    time_group_flag = True
                 else:
                     sql_seqs.append(seq)
             sql = "\n".join(sql_seqs)
 
+            if time_group_flag:
+                temp_sql = sql
+                min_sql_seqs = []
+                max_sql_seqs = []
+                temp_seqs = temp_sql.split("\n")
+                for seq in temp_seqs:
+                    if seq.find("SELECT") >= 0:
+                        min_seq = seq
+                        max_seq = seq
+                        aggregate_functions = ["count", "max_value", "min_value", "sum", "avg"]
+                        for func in aggregate_functions:
+                            max_seq = re.sub(r"\b" + func + r"\b", "MAX_TIME",  max_seq)
+                            min_seq = re.sub(r"\b" + func + r"\b", "MIN_TIME",  min_seq)
```

Comment:
This should be case insensitive.
```suggestion
                        aggregate_functions = ["count", "max_value", "min_value", "sum", "avg"]
                        for func in aggregate_functions:
                            max_seq = re.sub(r"\b" + func + r"\b", "MAX_TIME",  max_seq, flags=re.IGNORECASE)
                            min_seq = re.sub(r"\b" + func + r"\b", "MIN_TIME",  min_seq, flags=re.IGNORECASE)
```